Mogakco/Sources/Presentation/Chat/ViewController/ChatViewController.swift

```diff
@@ -85,18 +126,73 @@ final class ChatViewController: UICollectionViewController {
     
     private func layoutCollectionView() {
         collectionView.snp.makeConstraints {
-            $0.top.left.right.equalToSuperview()
-            $0.bottom.equalToSuperview().inset(90)
+            $0.top.left.bottom.right.equalToSuperview()
         }
         collectionView.register(ChatCell.self, forCellWithReuseIdentifier: ChatCell.identifier)
         collectionView.alwaysBounceVertical = true
     }
     
+    func sidebarDidTap(row: ChatSidebarMenu) {
+        blackScreen.isHidden = true
+        blackScreen.frame = self.view.bounds
+        
+        UIView.animate(withDuration: 0.3) {
+            self.sidebarView.frame = CGRect(
+                x: self.view.frame.width,
+                y: 0,
+                width: 0,
+                height: self.sidebarView.frame.height
+            )
+        }
+        
+        switch row {
+        case .studyInfo:
+            print("1")
+        case .exitStudy:
+            print("2")
+        case .showMember:
+            print("3")
+        }
+    }
+    
     @objc private func backButtonDidTap() {
         self.navigationController?.popViewController(animated: true)
     }
     
     @objc private func studyInfoDidTap() {
+        blackScreen.isHidden = false
+        
+        UIView.animate(
+            withDuration: 0.3,
+            animations: {
+                self.sidebarView.frame = CGRect(
+                    x: self.view.frame.width * (2 / 3),
+                    y: 0,
+                    width: 250,
+                    height: self.sidebarView.frame.height)
+            },
+            completion: { _ in
+                self.blackScreen.frame = CGRect(
+                    x: 0,
+                    y: 0,
+                    width: 300,
+                    height: self.view.bounds.height)
+            }
+        )
```

Comment:
```suggestion
        UIView.animate(
            withDuration: 0.3,
            animations: {
                self.sidebarView.frame = CGRect(
                    x: self.view.frame.width * (2 / 3),
                    y: 0,
                    width: self.view.frame.width * (1 / 3),
                    height: self.sidebarView.frame.height)
            },
            completion: { _ in
                self.blackScreen.frame = CGRect(
                    x: 0,
                    y: 0,
                    width: self.view.frame.width * (2 / 3),
                    height: self.view.bounds.height)
            }
        )
```

viewController의 1/3 만큼만 SidebarView가 나타나는 거라면 width도 상대값으로 맞춰주면 좋을 것 같아요!

Reply:
넵 반영하겠습니다!